{
  "dataType": "CVE_RECORD",
  "cveMetadata": {
    "state": "PUBLISHED",
    "cveId": "CVE-2024-41990",
    "assignerOrgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
    "assignerShortName": "mitre",
    "dateUpdated": "2024-09-18T14:57:21.043Z",
    "dateReserved": "2024-07-25T00:00:00",
    "datePublished": "2024-08-07T00:00:00"
  },
  "containers": {
    "cna": {
      "providerMetadata": {
        "orgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
        "shortName": "mitre",
        "dateUpdated": "2024-08-07T14:11:45.667382"
      },
      "descriptions": [
        {
          "lang": "en",
          "value": "An issue was discovered in Django 5.0 before 5.0.8 and 4.2 before 4.2.15. The urlize() and urlizetrunc() template filters are subject to a potential denial-of-service attack via very large inputs with a specific sequence of characters."
        }
      ],
      "affected": [
        {
          "vendor": "n/a",
          "product": "n/a",
          "versions": [
            {
              "version": "n/a",
              "status": "affected"
            }
          ]
        }
      ],
      "references": [
        {
          "url": "https://docs.djangoproject.com/en/dev/releases/security/"
        },
        {
          "url": "https://groups.google.com/forum/#%21forum/django-announce"
        },
        {
          "url": "https://www.djangoproject.com/weblog/2024/aug/06/security-releases/"
        }
      ],
      "problemTypes": [
        {
          "descriptions": [
            {
              "type": "text",
              "lang": "en",
              "description": "n/a"
            }
          ]
        }
      ]
    },
    "adp": [
      {
        "title": "CISA ADP Vulnrichment",
        "metrics": [
          {
            "cvssV3_1": {
              "scope": "UNCHANGED",
              "version": "3.1",
              "baseScore": 7.5,
              "attackVector": "NETWORK",
              "baseSeverity": "HIGH",
              "vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
              "integrityImpact": "NONE",
              "userInteraction": "NONE",
              "attackComplexity": "LOW",
              "availabilityImpact": "HIGH",
              "privilegesRequired": "NONE",
              "confidentialityImpact": "NONE"
            }
          },
          {
            "other": {
              "type": "ssvc",
              "content": {
                "id": "CVE-2024-41990",
                "role": "CISA Coordinator",
                "options": [
                  {
                    "Exploitation": "none"
                  },
                  {
                    "Automatable": "yes"
                  },
                  {
                    "Technical Impact": "partial"
                  }
                ],
                "version": "2.0.3",
                "timestamp": "2024-08-07T15:20:51.732074Z"
              }
            }
          }
        ],
        "affected": [
          {
            "cpes": [
              "cpe:2.3:a:djangoproject:django:*:*:*:*:*:*:*:*"
            ],
            "vendor": "djangoproject",
            "product": "django",
            "versions": [
              {
                "status": "affected",
                "version": "4.2",
                "lessThan": "4.2.15",
                "versionType": "custom"
              },
              {
                "status": "affected",
                "version": "5.0",
                "lessThan": "5.0.8",
                "versionType": "custom"
              }
            ],
            "defaultStatus": "unknown"
          },
          {
            "cpes": [
              "cpe:2.3:a:djangoproject:django:*:*:*:*:*:*:*:*"
            ],
            "vendor": "djangoproject",
            "product": "django",
            "versions": [
              {
                "status": "affected",
                "version": "pkg:pypi/django@4.2",
                "lessThan": "pkg:pypi/django@4.2.15",
                "versionType": "custom"
              },
              {
                "status": "affected",
                "version": "pkg:pypi/django@5.0",
                "lessThan": "pkg:pypi/django@5.0.8",
                "versionType": "custom"
              }
            ],
            "defaultStatus": "unknown"
          }
        ],
        "problemTypes": [
          {
            "descriptions": [
              {
                "lang": "en",
                "type": "CWE",
                "cweId": "CWE-130",
                "description": "CWE-130 Improper Handling of Length Parameter Inconsistency"
              }
            ]
          }
        ],
        "providerMetadata": {
          "orgId": "134c704f-9b21-4f2e-91b3-4a467353bcc0",
          "shortName": "CISA-ADP",
          "dateUpdated": "2024-08-07T15:20:11.171Z"
        }
      }
    ]
  },
  "dataVersion": "5.1"
}